src/mame/drivers/nmk16.cpp

```diff
@@ -7592,93 +7566,115 @@ void afega_state::init_redhawki()
 }
 
 ROM_START( redhawki )
-	ROM_REGION( 0x80000, "maincpu", 0 )     /* 68000 Code */
+	ROM_REGION( 0x80000, "maincpu", 0 )     // 68000 Code
 	ROM_LOAD16_BYTE( "rhit-2.bin", 0x000000, 0x020000, CRC(30cade0e) SHA1(2123ca858bcaed5165739107ccc2830561af0b38) )
 	ROM_LOAD16_BYTE( "rhit-3.bin", 0x000001, 0x020000, CRC(37dbb3c2) SHA1(d1f8258f357b885d38f87d288f98046dbd7d56aa) )
 
-	ROM_REGION( 0x10000, "audiocpu", 0 )        /* Z80 Code */
+	ROM_REGION( 0x10000, "audiocpu", 0 )        // Z80 Code
 	ROM_LOAD( "1.bin", 0x00000, 0x10000, CRC(5d8cf28e) SHA1(2a440bf5136f95af137b6688e566a14e65be94b1) )
 
-	ROM_REGION( 0x100000, "sprites", 0 )   /* Sprites, 16x16x4 */
+	ROM_REGION( 0x100000, "sprites", 0 )   // Sprites, 16x16x4
 	ROM_LOAD16_BYTE( "rhit-6.bin", 0x000001, 0x080000, CRC(7cbd5c60) SHA1(69bd728861ea5a02f514d5aed837b549f3c86019) )
 	ROM_LOAD16_BYTE( "rhit-7.bin", 0x000000, 0x080000, CRC(bcb367c7) SHA1(a8f0527bf75a227cdfd98385549892fb16330aea) )
 
-	ROM_REGION( 0x080000, "bgtile", 0 )   /* Layer 0, 16x16x8 */
+	ROM_REGION( 0x080000, "bgtile", 0 )   // Layer 0, 16x16x8
 	ROM_LOAD( "rhit-4.bin", 0x000000, 0x080000, CRC(aafb3cc4) SHA1(b5f6608c1e05470fdfb22e0a35a8a74974c4d3cf) )
 
-	ROM_REGION( 0x00100, "fgtile", ROMREGION_ERASEFF )    /* Layer 1, 8x8x4 */
+	ROM_REGION( 0x00100, "fgtile", ROMREGION_ERASEFF )    // Layer 1, 8x8x4
 	// Unused
 
-	ROM_REGION( 0x40000, "oki1", 0 )    /* Samples */
+	ROM_REGION( 0x40000, "oki1", 0 )    // Samples
 	ROM_LOAD( "5", 0x00000, 0x40000, CRC(e911ce33) SHA1(a29c4dea98a22235122303325c63c15fadd3431d) )
 ROM_END
 
+ROM_START( redhawks )
+	ROM_REGION( 0x80000, "maincpu", 0 )     // 68000 Code
+	ROM_LOAD16_BYTE( "2.hex", 0x000000, 0x020000, CRC(8b427ef8) SHA1(ba615b1a5ed9c1a97bb0b6d121a0d752d138adee) )
+	ROM_LOAD16_BYTE( "3.hex", 0x000001, 0x020000, CRC(117e3813) SHA1(415b115a96f139094b5927637c5ec8438cc9bd44) )
+
+	ROM_REGION( 0x10000, "audiocpu", 0 )        // Z80 Code
+	ROM_LOAD( "1.hex", 0x00000, 0x10000, CRC(5d8cf28e) SHA1(2a440bf5136f95af137b6688e566a14e65be94b1) )
+
+	ROM_REGION( 0x100000, "sprites", 0 )   // Sprites, 16x16x4
+	ROM_LOAD16_BYTE( "6.hex", 0x000001, 0x080000, CRC(aa6564e6) SHA1(f8335cddc0bb0674e86ccaca079ca828ed7a6790) )
+	ROM_LOAD16_BYTE( "7.hex", 0x000000, 0x080000, CRC(5c5b5fa1) SHA1(41946d763f9d72a6322a2f7e3c54a9f6114afe01) )
+
+	ROM_REGION( 0x080000, "bgtile", 0 )   // Layer 0, 16x16x8
+	ROM_LOAD( "4.hex", 0x000000, 0x080000, CRC(03a8d952) SHA1(44252f90e21d6f3841bcdcdac0aba318f94e33b0) )
+
+	ROM_REGION( 0x00100, "fgtile", ROMREGION_ERASEFF )    // Layer 1, 8x8x4
+	// Unused
+
+	ROM_REGION( 0x40000, "oki1", 0 )    // Samples
+	ROM_LOAD( "5.hex", 0x00000, 0x40000, CRC(e911ce33) SHA1(a29c4dea98a22235122303325c63c15fadd3431d) )
```

Comment:
Why the `hex` extension?  These aren’t actually Intel hex format are they?  What are the labels or locations on the board?

Reply:
Victor named them as hex, but yes, it's confusing, I've renamed them to bin. The labels were just numbers.